ocs_ci/deployment/deployment.py

```diff
@@ -1476,23 +1473,6 @@ def configure_mirror_peer(self):
         config.switch_acm_ctx()
         run_cmd(f"oc create -f {mirror_peer_yaml.name}")
```

Comment:
There is a change in MirrorPeer template for 4.10, we need to include that for 4.10 changes.

Reply:
@sidhant-agrawal ack, could you point me to new template so that I can add it as part of this PR

Reply:
You can refer to Section 5.2 of the [docs](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/configuring_openshift_data_foundation_for_regional-dr_with_advanced_cluster_management#creating-mirror-peer-on-hub-cluster_rhodf)

Reply:
thanks, Done